src/maml/algorithm/meta_learner.py

```diff
@@ -149,7 +149,7 @@ def meta_update(self):
                 # Compute Meta-loss and backpropagate it through the gradient steps.
                 # Losses across all of the batch tasks are cumulated
                 # until `self.outer_optimizer.step()`
-                policy_loss = self.agent.compute_loss(inner_policy, validation_batch, is_metaloss=True)
+                policy_loss = self.agent.compute_loss(inner_policy, validation_batch, is_meta_loss=True)
```

Comment:
간단한거긴 한데, train_batch와 동일한 글자수로 valid_batch로 바꾸면 어떨까요?
```suggestion
                policy_loss = self.agent.compute_loss(inner_policy, valid_batch, is_meta_loss=True)
```

Reply:
그렇겠네요! 반영하였습니다